src/renderer/gltf/index.ts

```diff
@@ -233,8 +223,7 @@ function makeBuffer(
   gl: WebGL2RenderingContext,
```

Comment:
I suspect this function no longer needs to take the context?

Reply:
Oh wait, just for `accessorComponentTypeToGLType`, which is a little unfortunate, since the values are ultimately constants. I think it's okay for now but we should eventually factor that out.